backend/src/main/java/clf/integra/backend/service/UserService.java

```diff
@@ -3,10 +3,21 @@
 import clf.integra.backend.repository.UserRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.UUID;
 
 @Service
 @RequiredArgsConstructor
 public class UserService {
 
     private final UserRepository userRepository;
+
+    public Double getBalance(UUID id) {
+        try {
+            return userRepository.getBalance(id);
```

Comment:
This is not a good validation approach with try { } catch {} (at least in this case), I would recommend checking if the user is null in the repo and return null then -> and here just return the repository method